table.csv

```diff
@@ -111,6 +111,7 @@ ws,                             multiaddr,      0x01dd,         permanent,
 wss,                            multiaddr,      0x01de,         permanent,
 p2p-websocket-star,             multiaddr,      0x01df,         permanent,
 http,                           multiaddr,      0x01e0,         draft,
+kcp,                            multiaddr,      0x01e1,         draft,
```

Comment:
This is the text in the repository.

> A Crypto-Secure, Production-Grade Reliable-UDP Library for golang with FEC

It has CFB encryption mode built into it also, I'm not sure how to handle that, I assume that if the codec type exists it can just be implemented that way (as in, not part of the constructors), and uses RS FEC, also I haven't checked if this type of processing is already covered in the codecs list?